templates/sag-cc-builder-dev/README.md

```diff
@@ -32,14 +34,8 @@ All supported Windows and UNIX platforms.
 
 ## Running as a standalone composite template
 
-To add product, fix, and asset repositories in Command Central:
+To add internal repositories to use with Command Central Docker builder:
```

Comment:
as above:
internal product, fix and asset

Reply:
Updated here as well.